### PR TITLE
Implemented cards for Moscovia monsters

### PR DESCRIPTION
**Pull request by secretdataz**
* **Addressed Issue(s)**: None
* **Server Mode**: Renewal
* **Description of Pull Request**: 
Implemented cards for monsters in Moscovia.
Sources:
https://www.divine-pride.net/database/item/27157
https://www.divine-pride.net/database/item/27158
https://www.divine-pride.net/database/item/27159
https://www.divine-pride.net/database/item/27160
https://www.divine-pride.net/database/item/27161
https://www.divine-pride.net/database/item/27162

### CODE REVIEW

**Comment by Lemongrass3110** [comment]
https://ci.appveyor.com/project/rAthenaAPI/rathena/build/feature/moscovia_cards-1860/job/742cgd9ycavcdh1v#L703

File: db/re/item_db.txt [hunk 1]
```diff
@@ -11424,6 +11424,12 @@
 27150,Toxic_Enchanted_Chimera_Card,Toxic Enchanted Chimera Card,6,20,,10,,,,,,,,64,,,,,{ bonus bMaxHP,1000; bonus bMaxSP,200; if(getrefine() >= 12){ bonus bMaxHPrate,15; bonus bMaxSPrate,15; } },{},{}
 27151,Heart_Hunter_Evil_Card,Heart Hunter Evil Card,6,20,,10,,,,,,,,16,,,,,{ bonus2 bAddEffWhenHit,SC_FREEZING,1000 + (readparam(bInt) >= 110 ? 5 : 0); bonus bMdef,5; },{},{}
 27152,Cutie_Card,Cutie Card,6,20,,10,,,,,,,,64,,,,,{ bonus bMaxHPrate,10; .@r = getrefine()/2; bonus bAgi,.@r; bonus bAspdRate,.@r; },{},{}
+27157,Wood_Goblin_Card,Wood Goblin Card,6,20,,10,,,,,,,,64,,,,,{ .@r = getrefine(); bonus2 bAddEle,Ele_Water,(.@r >= 9 ? 10 : 5); bonus2 bAddEle,Ele_Earth,(.@r >= 9 ? 10 : 5); },{},{}
+27158,Les_Card,Les Card,6,20,,10,,,,,,,,32,,,,,{ bonus2 bSubEle,Ele_Wind,30; },{},{}
+27159,Uzhas_Card,Uzhas Card,6,20,,10,,,,,,,,769,,,,,{ bonus2 bMagicAddRace,RC_Demon,(getrefine() >= 9 ? 15 : 10); },{},{}
+27160,Vavayaga_Card,Vavayaga Card,6,20,,10,,,,,,,,64,,,,,{ bonus2 bFlee,getrefine()*2; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: `bonus bFlee,n;`

File: db/re/item_db.txt [hunk 2]
```diff
@@ -11424,6 +11424,12 @@
 27150,Toxic_Enchanted_Chimera_Card,Toxic Enchanted Chimera Card,6,20,,10,,,,,,,,64,,,,,{ bonus bMaxHP,1000; bonus bMaxSP,200; if(getrefine() >= 12){ bonus bMaxHPrate,15; bonus bMaxSPrate,15; } },{},{}
 27151,Heart_Hunter_Evil_Card,Heart Hunter Evil Card,6,20,,10,,,,,,,,16,,,,,{ bonus2 bAddEffWhenHit,SC_FREEZING,1000 + (readparam(bInt) >= 110 ? 5 : 0); bonus bMdef,5; },{},{}
 27152,Cutie_Card,Cutie Card,6,20,,10,,,,,,,,64,,,,,{ bonus bMaxHPrate,10; .@r = getrefine()/2; bonus bAgi,.@r; bonus bAspdRate,.@r; },{},{}
+27157,Wood_Goblin_Card,Wood Goblin Card,6,20,,10,,,,,,,,64,,,,,{ .@r = getrefine(); bonus2 bAddEle,Ele_Water,(.@r >= 9 ? 10 : 5); bonus2 bAddEle,Ele_Earth,(.@r >= 9 ? 10 : 5); },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: `Increase damage taken from water and earth properties monsters by 5%.` is missing

Comment (Atemo) [reply db/re/item_db.txt]: Should be `Compound on: Armor`
```
Loc: Equipment's placement. Values are:
	2^4      016 = Armor
```

**Pull request merged**